applications/Unity.GrantManager/src/Unity.GrantManager.Application/Assessments/AssessmentAppService.cs

```diff
@@ -188,5 +189,38 @@ private static OperationAuthorizationRequirement GetActionAuthorizationRequireme
             return new OperationAuthorizationRequirement { Name = $"{GrantApplicationPermissions.Assessments.Default}.{triggerAction}" };
         }
         #endregion ASSESSMENT WORKFLOW
+
+        public async Task UpdateAssessmentScore(AssessmentScoresDto dto)
+        {
+            try
+            {
+                var assessment = await _assessmentRepository.GetAsync(dto.AssessmentId);
+                if (assessment != null)
+                {
+                    if(CurrentUser.GetId() != assessment.AssessorId)
```

Comment:
These are business rules that need to be moved into the domain layer, and possibly the state machine

Reply:
> These are business rules that need to be moved into the domain layer, and possibly the state machine

hi @AndreGAot -- can i remove this logic for now and rework on it in a separate user story for "assessment score state machine"?